packages/web3-providers/src/NFTScan/helpers/EVM.ts

```diff
@@ -71,6 +71,75 @@ export async function fetchFromNFTScanV2<T>(chainId: ChainId, pathname: string,
     return json as T
 }
 
+export async function postNFTScanRestFulAPI<T>(pathname: string, body: string, init?: RequestInit) {
+    const response = await fetch(urlcat(NFTSCAN_RESTFUL_API, pathname), {
+        ...init,
+        method: 'POST',
+        headers: {
+            'content-type': 'application/json',
+            ...init?.headers,
+        },
+        cache: 'no-cache',
+        body,
+    })
+
+    const json = await response.json()
+    return json as T
+}
+
+export async function fetchFromNFTScanRestFulAPI<T>(pathname: string, init?: RequestInit) {
+    const response = await fetch(urlcat(NFTSCAN_RESTFUL_API, pathname), {
+        ...init,
+        method: 'GET',
+        headers: {
+            'content-type': 'application/json',
+            ...init?.headers,
+        },
+        cache: 'no-cache',
+    })
+
+    const json = await response.json()
+    return json as T
+}
+
+function resolveNFTScanAPIChain(chainId: ChainId): string {
+    switch (chainId) {
```

Comment:
Read the `shortName` of `NETWORK_DESCRIPTORS`.

Reply:
Not match.